### PR TITLE
UPSTREAM: <carry>: Change upstream OWNERS

### PR DESCRIPTION
**Pull request by Fedosin**
This commit changes owners of the project to people from OpenShift.

### CODE REVIEW

**Comment by JoelSpeed** [comment]
For now can we enforce git history commits with our standard prefix to make sure rebases can be smoother later?

File: OWNERS [hunk 1]
```diff
@@ -1,5 +1,7 @@
 approvers:
-- aoxn
-- cheyang 
-- xlgao-zju
-- gujingit
+- JoelSpeed
+- Danil-Grigorev
+- elmiko
```
[the remark this answers is not in the document]
Comment (Fedosin) [reply OWNERS:10]: done

**Comment by Fedosin** [comment]
> For now can we enforce git history commits with our standard prefix to make sure rebases can be smoother later?

do you mean `UPSTREAM: <carry>: `?

**Comment by JoelSpeed** [comment]
Yeah that's what I mean, would prefer to see the commit messages with the prefixes for now, preferably enforced by CI, then we can work out if we need them/drop it at a later date

**Review by elmiko — APPROVED** [review]
lgtm, thanks for the updates @Fedosin !

**Pull request merged**